examples/06_benchmarks/movielens.ipynb

```diff
@@ -76,18 +76,18 @@
      "name": "stdout",
```

Comment:
<p>since we are removing the common epochs, can you remove this parameter and add the value on each dictionary? I think this will reduce the confusion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
updated!